nf_core/subworkflow-template/tests/main.nf

```diff
@@ -4,7 +4,7 @@ nextflow.enable.dsl = 2
 
 include { {{ subworkflow_name|upper }} } from '../../../../subworkflows/{{ org }}/{{ subworkflow_dir }}/main.nf'
 
-workflow test_{{ subworkflow_name }} {
+workflow test_{{ component_name_underscore }} {
```

Comment:
Yes, the template is filled in the `SubworkflowCreate`  class (which is derived from `ComponentCreate`). Neither of these classes have an attribute `subworkflow_name` and the test initially failed because the workflow was named `test_`. It's the same in `nf_core/module-template/tests/main.nf`.